sessionctx/variable/varsutil.go

```diff
@@ -660,6 +660,15 @@ func ValidateSetSystemVar(vars *SessionVars, name string, value string) (string,
 			}
 		}
 		return formatVal, nil
+	case TiDBMetricSchemaStep, TiDBMetricSchemaRangeDuration:
+		v, err := strconv.ParseInt(value, 10, 64)
+		if err != nil {
+			return value, ErrWrongValueForVar.GenWithStackByArgs(name)
```

Comment:
This error needs 2 args, the `value` is also needed.

Reply:
great, done.